core/tests/protractor_utils/TopicsAndSkillsDashboardPage.js

```diff
@@ -138,15 +138,33 @@ var TopicsAndSkillsDashboardPage = function() {
     });
   };
 
-  this.createTopic = function(title) {
-    waitFor.elementToBeClickable(
-      createTopicButton,
-      'Create Topic button takes too long to be clickable');
-    createTopicButton.click();
+  this.createTopic = function(topicName, shouldCloseTopicEditor) {
```

Comment:
@U8NWXD I've added 2 different steps to handle tab management, one in this `createTopic` function and the other in the `createSkillWithDescriptionAndExplanation` function, can you take a look and see which one would be better?
Based on what I observed, this one in `createTopic` is faster, but the one in `createSkill` might be more reliable as we are checking for browser title.

Reply:
@aks681 I actually think the one in `createTopic` could be more reliable depending on how handle equality is evaluated. Would two tabs with the same title have handles that are considered unequal? That would be my guess, and if so, then the way in `createTopic` would work even if the tabs have the same title. Does that make sense? I don't see a scenario where the `createTopic` way would fail but the `createSkillWithDescriptionAndExplanation` way would work. Do you see any?

Reply:
Yup, the handles are unique even if titles are the same and yeah I don't think the `createSkill` one would have any benefits over the `createTopic` one unless multiple tabs get created at the same time which wouldn't happen.

Done, changed both to the one in `createTopic`.

Reply:
Looks good!